client-app/package.json

```diff
@@ -1,95 +1,94 @@
 {
-  "name": "my-app",
```

Comment:
@Gayatrinj `package.json` and `package-lock.json` file should not auto formatted by prettier as you can see [here ](https://github.com/oss-slu/material-donor-mutual-assist/blob/main/.prettierignore)that these files are ignored by prettier. Please do not push any trivial changes to `package.json` and `package-lock.json` file unless you have some new dependency added to it.

Reply:
Please revert changes to both of these files and push only if you have any new dependency added